storage/src/checker/mod.rs

```diff
@@ -37,6 +37,10 @@ impl<S: WritableStorage> NodeStore<Committed, S> {
     /// Panics if the header has too many free lists, which can never happen since freelists have a fixed size.
     // TODO: report all errors, not just the first one
     pub fn check(&self, opt: CheckOpt) -> Result<(), CheckerError> {
+        if cfg!(feature = "ethhash") {
+            unimplemented!("ethhash is not supported yet");
+        }
```

Comment:
This shouldn't panic here as it will cause the tests to fail. broken tests should be annotated with `#[ignore]` and/or this function should not be compiled in for ethhash.

Reply:
Let me ignore the tests for ethhash - this is temporary as we are currently working on a fix for ethhash. It may take a while so we just want to block it for now.